WooCommerce/Classes/Authentication/Navigation Exceptions/WrongAccountErrorViewModel.swift

```diff
@@ -106,6 +152,151 @@ final class WrongAccountErrorViewModel: ULAccountMismatchViewModel {
     }
 }
 
+// MARK: - Private helpers
+private extension WrongAccountErrorViewModel {
+    /// Fetches the site info and show the primary button if the site is self-hosted.
+    /// This will enables Jetpack connection support for self-hosted sites.
```

Comment:
❓ 
> This will enables Jetpack connection support for self-hosted sites.

Does this mean that tapping on the primary button will enable JCP for self-hosted sites? 

I want to clarify this because I hope that this `fetchSiteInfo` method isn't doing anything related to enabling JCP on its own. 🤔

Reply:
I meant that the returned information will determine whether the Connect Jetpack button should be visible for handling Jetpack connection. This has nothing to do with JCP 😅. Updated the comment in 071bcd9be1954dfca147045835e2351e997693e2.